proto/tasks.proto

```diff
@@ -9,6 +9,8 @@ import "common.proto";
 service Tasks {
     rpc WatchTasks(TasksRequest) returns (stream TaskUpdate) {}
     rpc WatchTaskDetails(DetailsRequest) returns (stream TaskDetails) {}
+    rpc Pause(PauseRequest) returns (PauseResponse) {}
+    rpc Resume(ResumeRequest) returns (ResumeResponse) {}
 }
```

Comment:
Hmm, as per my comment above, I think if we want to implement pausing in the aggregator rather than in the UI, we would want to have a separate temporality for each client connection, rather than adding new RPCs that globally mutate whether the app is paused or resumed.

IMO, the best way to do this would be to turn the `WatchTasks` and `WatchTaskDetails` RPCs into bidirectional streaming RPCs, like
```suggestion
service Tasks {
    rpc WatchTasks(stream WatchCommand) returns (stream TaskUpdate) {}
    rpc WatchTaskDetails(stream WatchCommand) returns (stream TaskDetails) {}
}

message WatchCommand {
    enum Kind {
        PAUSE = 0;
        RESUME = 1;
    } 
    Kind command_kind = 1;
}
```
or something? Then, when a `PAUSE` message is sent on a given RPC, the aggregator would pause sending updates to that client until it receives a `RESUME` command from the same client.